Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionContextCocoa.mm

```diff
@@ -1218,6 +1230,17 @@ static _WKWebExtensionContextError toAPI(WebExtensionContext::Error error)
     return tab;
 }
 
+RefPtr<WebExtensionTab> WebExtensionContext::getTab(WebPageProxyIdentifier webPageProxyIdentifier, std::optional<WebExtensionTabIdentifier> identifier)
+{
+    if (!identifier) {
```

Comment:
I think it might be clearer to switch these conditions, and have the first block be:

`if (identifier)`...

Reply:
I agree. Will do.